### PR TITLE
Model serving bean stalk ec2

### PR DESCRIPTION
**Pull request by JonTanS**
Updated with a AWS BeanStalk Model Serving Example.

Uses the barebones functionality of Qing's web example and sets up an API endpoint that will do inference on a passed in encoded string.

### CODE REVIEW

File: beanstalk-ec2-model-serving/src/main/java/com/example/djlInferenceBeanStalk/Handler.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+     * @return Returns the top 5 probable items from the model output
+     * @throws IOException
+     */
+    @PostMapping(value = "/inference")
+    public byte[] handleRequest(@RequestBody Request inputRequest) throws IOException {
+        // Grabs the string image from the input
+        String imageInput = inputRequest.getImageData();
+        try {
+            // Decodes the string and transforms it into an image
+            byte[] imgBytes = Base64.getDecoder().decode(imageInput);
```
Comment (frankfliu) [inline beanstalk-ec2-model-serving/src/main/java/com/example/djlInferenceBeanStalk/Handler.java:74]: I think we should take binary image instead.

File: beanstalk-ec2-model-serving/README.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
+# Model Serving on AWS BeanStalk EC2
+
+## Overview
+DJL makes it easy to deploy your deep learning models on AWS Beanstalk.
+Deploying on AWS Beanstalk is beneficial where there are low latency requirements 
+for all inference requests that can happen at any time. 
+The cost on the other hand will be higher because the EC2 instance that is hosting
```
Comment (zachgk) [inline beanstalk-ec2-model-serving/README.md:7]: cost is higher than serverless like lambda

File: beanstalk-ec2-model-serving/src/main/java/com/example/djlInferenceBeanStalk/Handler.java [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+@RestController
+public class Handler {
+    Logger logger = Logger.getLogger(Handler.class.getName());
+
+    private static final Gson GSON = new GsonBuilder().setPrettyPrinting().create();
+
+    static {
+        // DJL saves model and native libraries in cache folder.
+        // In AWS-Lambda only /tmp folder is writable.
```
Comment (zachgk) [inline beanstalk-ec2-model-serving/src/main/java/com/example/djlInferenceBeanStalk/Handler.java:57]: In Lambda?

File: beanstalk-ec2-model-serving/src/main/resources/Images/Smiley_Face.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAYAAACqaXHeAAADGElEQVR4Xu1bv0uyQRz/CKkJCQoaCk6NIUIQSBhIjS2CuOhkNtTgn5BL/g0NRZAuSoODQ2PhYKAIDgquDYEgNihBWmq+nPC8vL2kdw+eFz7ebXrf53vfz+f767zn1AEYY4WHThIgI0CmgKwBK1wDIbQIms1mvL29Ufne3t5Go9GgyvEQEEbAwcEBHh8fodORJWeP8XjMJEfTwzIvjIDDw0M8PDwwAVtpAnZ2dlCtVpmIYvEwTUZYBPh8PhSLRSqwXq+H9fV1qhwNGOu8MAJYU0CzBKytrWEwGFA92+/3YTQaqXKsHqbJCYsAk8mE9/d3KjDNEkC8SsCRSBiNRlMdc3JygpubGypRNM+yzguLAGIQaW+RSATZbHaqfbu7u6hUKtol4Pj4GKlUaioBfr8fhUJhdQlQIoVlx8ga5rPkhKdANBpFOp2eaTtJFbIX+Pj44IFxpg7hBMRiMdze3s40qlarwePxLBw8WUAoAZ+fn7Db7eh2u0LAsSwilAAWg0TLLC0BpE6QSLJYLHNxtpQEGAyGSYE8PT3F9fX17xOQyWQQDoeF9W7i/UkBYzhcobHDJQK8Xi9KpRL0ej2GwyFtzbnmeYLn2gV4G/YTS4lEAhcXF3h+fsbW1tZcRCoPc4kAoszpdKLZbKLVasHhcHAx7l8lwWAQuVwOnU4HVquVm35uBBCLkskkzs/PueWngvLo6Aj39/fc9XJNAcVY5UyPfI7H47i8vJzLWy8vL3C5XAsBvxACFLRfX19/q/TZ2Rmurq6YiQgEAri7u5ucDJHx9PSE/f195ufVCHJNgf8XdrvdqNfr374mxXJzc3PyHdnIkGMyMmw2G9rt9jfZ19fXydZ5kWOhBCiGkzdC5MVIPp+nYiEE7e3toVwuU2V5CAghYJqhoVAIGxsbk1Sh/ULkAfYnHb9KwKJAqdErCZBXZOQVGXlFRl6RUVM1tSYru4DsArILyC4gu4DWKrsaPLILyC4gu4DsArILqKmaWpOVXUB2Aa3FtEo89H8vqFS4bOKSgGXzGG97ZQTwZnTZ9MkIWDaP8bZ35SPgD3PSDxkcJ3pXAAAAAElFTkSuQmCC
```
Comment (zachgk) [inline beanstalk-ec2-model-serving/src/main/resources/Images/Smiley_Face.txt:1]: Can you put the images on S3 instead of checking them into git

Comment (JonTanS) [reply beanstalk-ec2-model-serving/src/main/resources/Images/Smiley_Face.txt:1]: Should I just put this in the public s3 where I think the hugging face model is hosted?

File: beanstalk-ec2-model-serving/src/main/java/com/example/djlInferenceBeanStalk/Handler.java (hunk 1 above)
Comment (JonTanS) [reply beanstalk-ec2-model-serving/src/main/java/com/example/djlInferenceBeanStalk/Handler.java:74]: Sure we can do either one I was just following Qing's example where he uses the string representation

**Pull request merged**